src/samples/OpenChannelSendAndReceiveVariousTypesOfFiles.js

```diff
@@ -323,10 +329,11 @@ const ChannelList = ({ channels, handleJoinChannel, toggleShowCreateChannel, han
 }
 
 
-const Channel = ({ currentlyJoinedChannel, handleLeaveChannel, children }) => {
+const Channel = ({ currentlyJoinedChannel, handleLeaveChannel, children, isImageSizeError }) => {
     if (currentlyJoinedChannel) {
         return <div className="channel">
             <ChannelHeader>{currentlyJoinedChannel.name}</ChannelHeader>
+            {isImageSizeError && <div className='image-size-error'>You subscription plan doesn't support the file size</div>}
```

Comment:
Any failure around sending a file will causes this error so seems like we cant be as explicit as "You subscription plan doesn't support the file size".

I would suggest either switch on the error message here https://github.com/sendbird/sendbird-chat-sample-react/pull/72/commits/ffad744afaa6e3c9843bd3f5b43a3a9d9804162b#diff-2c24dcf7a474cf2fa972ba61d356fa488d319b6b1d179f3b168c138b150a1495L194. Or if not possible make the error more generic